src/CMakeLists.txt

```diff
@@ -75,8 +77,9 @@ add_custom_target(runefu
     COMMAND perfgen "-s" "1" "--nohwcheck"
     COMMAND dream "-f" "../configs/DreamEndcap.json" "-s" "2" "--nohwcheck"
     COMMAND freia "-f" "../configs/freia.json" "-s" "2" "--nohwcheck" "-l" "Info" "--log_file" "deleteme_graylog.log"
-    COMMAND cspec "-f" "../configs/cspec.json" "-s" "2" "--nohwcheck"
+    COMMAND trex "-f" "../configs/tex.json" "-s" "2" "--nohwcheck"
     COMMAND ttlmonitor "-f" "../configs/freiamon.json" "-s" "2" "--nohwcheck" "--dumptofile" "deleteme_ttlmon_" "-t" "freia_beam_monitor"
     COMMAND nmx "-f" "../configs/nmx.json" "-s" "2"  "--nohwcheck"
-    DEPENDS copycfgs dream loki bifrost miracles freia cspec ttlmonitor nmx
+    COMMAND cspec "-f" "../configs/he3cpec.json" "-s" "2" "--nohwcheck" "--dumptofile" "deleteme_cspec.h5"
```

Comment:
I suggest keeping the old cspec.json name

Reply:
This is an oversight from the string replacement. I'll change it.